po/sl.po

```diff
@@ -0,0 +1,6238 @@
+msgid "ID"
+msgstr "ID"
+
+#: app/flatpak-builtins-build-export.c:75
+#, fuzzy
+msgid "Override the timestamp of the commit"
+msgstr "Preglasi časovni žig zaobvesti"
+
+#: app/flatpak-builtins-build-export.c:76
+#: app/flatpak-builtins-build-update-repo.c:74
+#: app/flatpak-builtins-remote-add.c:80 app/flatpak-builtins-remote-list.c:52
+#: app/flatpak-builtins-remote-modify.c:87
+#, fuzzy
+msgid "Collection ID"
+msgstr "ID zbirke"
+
+#: app/flatpak-builtins-build-export.c:469
+#, fuzzy, c-format
+msgid "WARNING: Error running desktop-file-validate: %s\n"
+msgstr "OPOZORILO: Napaka pri zagonu namizne datoteke: %s\n"
+
+#: app/flatpak-builtins-build-export.c:477
+#, fuzzy, c-format
+msgid "WARNING: Error reading from desktop-file-validate: %s\n"
+msgstr "OPOZORILO: Napaka pri branju iz namizne datoteke-validate: %s\n"
+
+#: app/flatpak-builtins-build-export.c:483
+#, fuzzy, c-format
+msgid "WARNING: Failed to validate desktop file %s: %s\n"
+msgstr "OPOZORILO: Preverjanje veljavnosti namizne datoteke %s: %s\n"
+
+#: app/flatpak-builtins-build-export.c:498
+#, fuzzy, c-format
+msgid "WARNING: Can't find Exec key in %s: %s\n"
+msgstr "OPOZORILO: Ključa Exec ni mogoče najti v %s: %s\n"
+
+#: app/flatpak-builtins-build-export.c:506
+#: app/flatpak-builtins-build-export.c:603
+#, fuzzy, c-format
+msgid "WARNING: Binary not found for Exec line in %s: %s\n"
+msgstr "OPOZORILO: Binarnega ni bilo mogoče najti za exec linijo v %s: %s\n"
+
+#: app/flatpak-builtins-build-export.c:513
+#, fuzzy, c-format
+msgid "WARNING: Icon not matching app id in %s: %s\n"
+msgstr "OPOZORILO: Ikona se ne ujema z ID-jem aplikacije v %s: %s\n"
+
+#: app/flatpak-builtins-build-export.c:541
+#, fuzzy, c-format
+msgid "WARNING: Icon referenced in desktop file but not exported: %s\n"
+msgstr ""
+"OPOZORILO: Ikona, sklicena v namizni datoteki, vendar ni izvožena: %s\n"
+
+#: app/flatpak-builtins-build-export.c:708
+#, fuzzy, c-format
+msgid "Invalid uri type %s, only http/https supported"
+msgstr "Neveljavna vrsta uri %s, podprta samo http/https"
+
+#: app/flatpak-builtins-build-export.c:726
+#, fuzzy, c-format
+msgid "Unable to find basename in %s, specify a name explicitly"
+msgstr "Ni mogoče najti osnovnega imena v %s, izrecno navedite ime"
+
+#: app/flatpak-builtins-build-export.c:735
+#, fuzzy
+msgid "No slashes allowed in extra data name"
+msgstr "V dodatnem podatkovnem imenu ni dovoljenih poševnic"
+
+#: app/flatpak-builtins-build-export.c:747
+#, fuzzy, c-format
+msgid "Invalid format for sha256 checksum: '%s'"
+msgstr "Neveljavna oblika za kontrolno vsoto sha256: \"%s\""
+
+#: app/flatpak-builtins-build-export.c:757
+#, fuzzy
+msgid "Extra data sizes of zero not supported"
+msgstr "Dodatne velikosti podatkov ničle niso podprte"
+
+#: app/flatpak-builtins-build-export.c:819
+#, fuzzy
+msgid ""
+"LOCATION DIRECTORY [BRANCH] - Create a repository from a build directory"
+msgstr ""
+"LOKACIJSKI IMENIK [PODRUŽNICA] - Ustvarite skladišče iz gradilnega imenika"
```

Comment:
You use `PODRUŽNICA` here and `VEJA` elsewhere, they should match as they are the same argument.